perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ExtSourcesManagerMethod.java

```diff
@@ -62,7 +62,7 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 
 		@Override
 		public ExtSource call(ApiCaller ac, Deserializer parms) throws PerunException {
-			return ac.getExtSourceById(parms.readInt("id"));
+			return ac.getExtSourcesManager().getExtSourceById(ac.getSession(), parms.readInt("id"));
```

Comment:
This is called by CLI when using tools to add/remove UES or set/remove their attributes. We should check, if user (SELF) can call these methods and if yes, we should enable SELF role in this methods policy.